hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -165,7 +165,11 @@
     FRONTEND_MESSAGE_PREVIEW_LOG_PERSIST_PERIOD("frontend.message.preview.log.persist.period.seconds", 30),
 
     FRONTEND_STARTUP_WAIT_KAFKA_ENABLED("frontend.startup.wait.kafka.enabled", false),
-    FRONTEND_STARTUP_WAIT_KAFKA_INTERVAL("frontend.startup.wait.kafka.interval", 5000L),
+
+    FRONTEND_KAFKA_HEALTH_CHECK_INTERVAL("frontend.startup.wait.kafka.interval", 5000L),
+    FRONTEND_KAFKA_HEALTH_CHECK_WAIT_TIMEOUT("frontend.startup.wait.kafka.timeout.ms", 24 * 60 * 60 * 1000L),
```

Comment:
Osobiście do definiowania takich wartości lubię używać `Duration.ofSeconds(5).toMillis()` i `Duration.ofDays(1).toMillis()`.
Łatwiej mi wtedy zrozumieć ile to czasu, a `toMillis()` (lub `toSeconds()` - zależy od potrzeb) pozwala to przekazać w miejsca, które wymagają liczby, a nie wspierają przekazywania `Duration`.

Bardziej skomplikowane wartości można zapisać w ten sposób
`Duration.ofDays(1).plusHours(3).plusMinutes(30).plusSeconds(15).toMillis()` lub zwięźlej `Duration.parse("P1DT3H30M15S").toMillis()`.

Wydaje mi się też, że w tym configu można by było po prostu zdefiniować obiekt typu `Duration` - `RetryPolicy` przyjmuje właśnie wartości tego typu.

Reply:
Yeah I totally agree with you. I just wanted to be consistent with actual convention. We already use this type of time defining in the whole configuration.

Reply:
we should do this improvement in different PR, I guess ?

Reply:
```
It also seems like this config does not need to have these timeouts defined as numbers - RetryPolicy's methods do accept Duration objects.
```
Since all remaining configs are defined as integers, this timeout should also follow this convention.

Changing configs that define milliseconds to duration values like `P1DT3H30M15S` will break the compatibility.